lib/main-menu.js

```diff
@@ -153,6 +153,19 @@ var view = {
   label: 'View',
   submenu: [
     {
+      label: 'Toggle Full Screen',
+      accelerator: (() => {
+        if (process.platform === 'darwin') return 'Ctrl+Command+F'
+        return 'F11'
+      })(),
+      click: (item, focusedWindow) => {
```

Comment:
@klauscfhq Thank you for this good information :smile: Year, my opinion is: `item` is used *inside* the menu, in other words, `item` is *not* used in our code. This is why I asked you to add a comment. Is this different from your opinion?